README.md

```diff
@@ -182,6 +182,69 @@ You can intermittently poll whether the operation has finished by using the `Pol
   }
 ```
 
+#### `Azure::Core::Context`
+
+Most Azure SDK Service Client methods accept an optional `Azure::Core::Context` parameter, which is used to enable cancellation of the operation or to
+establish an absolute deadline for the operation.
+
+This is useful when you want to assign a time limit on an operation to ensure that it completes in a "reasonable" timeframe. For instance, the 
+snippet below will cancel a blob client upload after 5 seconds.
+
+<!-- @insert_snippet: CreateBlobContext -->
+```cpp
+      Azure::Core::Context cancelledIn5s = Azure::Core::Context::CreateWithDeadline(
+          std::chrono::system_clock::now() + std::chrono::seconds(5));
+
+      auto containerClient = BlobContainerClient::CreateFromConnectionString(
+          GetConnectionString(), containerName + std::to_string(i));
+      containerClient.CreateIfNotExists({}, cancelledIn5s);
```

Comment:
Rnrelated: I wish we had a better dev exp when we have multiple optional parameters in C++. Having to pass "{}" for the options, to be able to provide context is a bit challenging to read.

Reply:
Unfortunately that's not the way C++ works. The only optional parameters are the *last* parameters in the method. And there's a parameter *after* the options parameter for CreateIfNotExists.